modules/ols-transcript-collection-overview.adoc

```diff
@@ -0,0 +1,18 @@
+// This module is used in the following assemblies:
+// about/ols-about-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: Concept
+[id="ols-transcript-collection-overview_{context}"]
+= Transcript collection overview 
+
+{ols-long} temporarily logs and stores complete transcripts of conversations that users have with the virtual assistant. This includes:
+
+* Queries from the user.
+
+* The complete message sent to the configured LLM provider, which includes system instructions, referenced documentation, and the user question.
+
+* The complete response from the LLM provider.
+
+Transcripts are associated with the cluster from which they originated, and {red-hat} can attribute specific clusters to specific customer accounts. Transcripts do not contain any information about which user submitted a question, and cannot be tied to any individual user.
```

Comment:
```suggestion
Transcripts originate from the cluster and are associated with the cluster. {red-hat} can assign specific clusters to specific customer accounts. Transcripts do not contain any information about users.
```

Same comments as above:

- In our docs, the term attribute stands for symbol, sign etc. So here, I suggest using "assign" or any similar term instead of "attribute".
- Consider removing the last sentence. "Feedback does not contain any information about users" sounds sufficient to me. because it also intends that it cannot be tied to users since that information is not collected at all.

Reply:
done